specs/nlp-go/_shared/contract.md

```
@@ -0,0 +1,333 @@
+
+---
+
+## 6. Streaming contract (`/go/studio/execute`)
+
+Server-Sent Events. Event shapes match `langwatch_nlp.studio.types.events.StudioServerEvent`:
+
+| event | data |
+|---|---|
+| `is_alive` | `{}` — heartbeat every `NLP_STREAM_HEARTBEAT_SECONDS` (default 15) |
+| `execution_state_change` | `{ trace_id, state: { status, nodes: { <node_id>: { status, inputs, outputs, error?, cost?, duration_ms? } } } }` |
+| `done` | `{ trace_id, status: "success"|"error", result }` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 2; Actual: 3; Too many cells, extra data will be missing

(MD056, table-column-count)